crates/ruma-events/src/sticker.rs

```diff
@@ -6,7 +6,33 @@ use ruma_common::OwnedMxcUri;
 use ruma_macros::EventContent;
 use serde::{Deserialize, Serialize};
 
-use crate::room::ImageInfo;
+#[cfg(not(feature = "compat-encrypted-stickers"))]
+use crate::room::EncryptedFile;
+use crate::room::{ImageInfo, MediaSource};
+
+/// The source of a sticker media file.
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
I believe that we should implement the same `Deserialize` as `MediaSource`, to have the same behavior.